engine/src/main/java/com/datatorrent/stram/StramLocalCluster.java

```diff
@@ -308,6 +313,24 @@ public StramLocalCluster(LogicalPlan dag) throws IOException, ClassNotFoundExcep
     this.dnmgr = new StreamingContainerManager(dag);
     this.umbilical = new UmbilicalProtocolLocalImpl();
 
+    Method addURL;
+    URLClassLoader loader;
+    try {
+      loader = (URLClassLoader)ClassLoader.getSystemClassLoader();
```

Comment:
There is difference between `loader.getClass()` and `URLClassLoader.class`. My proposal handles case when System class loader extends URLClassLoader.